docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -1232,6 +1232,27 @@ This can be used for joining a Teleport agent to a Teleport cluster which does n
   One option might be to use Teleport's built-in [ACME support](#acme) enable [cert-manager support](#highavailabilitycertmanager).
 </Admonition>
 
+## `existingDataVolume`
+
+| Type | Default value |
+| - | - |
+| `string` | `""` |
+
+When `existingDataVolume` is set to the name of an existing volume, the `/var/lib/teleport` mount will use this volume instead of creating a new `emptyDir` volume.
```

Comment:
I'm assuming that `if set` (below) means supplying a value other than the empty string?

Can you slightly rephrase this to better indicate that in the documentation - that the empty string is ignored? Specifically, that it will look for the directory `""` and use the default when it can't be found.

Otherwise, this is super helpful thanks!

Reply:
Not sure I follow, in Helm land it is pretty acceptable that a string is set if it is not `null` or `""`, so saying `if set` indicates it is anything but those values.